### PR TITLE
changes to pass pubmaticServer latency in psl field in logger call

### PR DESCRIPTION
**Pull request by pm-manasi-moghe**

### CODE REVIEW

File: src_new/bidManager.js [hunk 1]
```diff
@@ -333,7 +333,10 @@ exports.executeAnalyticsPixel = function(){ // TDD, i/o : done
 	util.forEachOnObject(window.PWT.bidMap, function (slotID, bmEntry) {
 		refThis.analyticalPixelCallback(slotID, bmEntry, impressionIDMap);
 	});
-
+	if (util.isOwnProperty(impressionIDMap, 'psl')) {
```
Comment (pm-harshad-mane) [inline src_new/bidManager.js]: indentation required

File: src_new/bidManager.js [hunk 2]
```diff
@@ -389,19 +393,22 @@ function analyticalPixelCallback(slotID, bmEntry, impressionIDMap) { // TDD, i/o
                 return;
             }
 
-						if (adapterID === "pubmaticServer") {
-								return;
-						}
-
             util.forEachOnObject(adapterEntry.bids, function(bidID, theBid) {
 
+				var endTime = theBid.getReceivedTime();
```
Comment (pm-harshad-mane) [inline src_new/bidManager.js]: indentation required

File: src_new/bidManager.js [hunk 3]
```diff
@@ -423,6 +430,9 @@ function analyticalPixelCallback(slotID, bmEntry, impressionIDMap) { // TDD, i/o
         });
 
         impressionIDMap[impressionID].push(slotObject);
+		if (pslTime !== undefined) {
```
Comment (pm-harshad-mane) [inline src_new/bidManager.js]: indentation required

File: src_new/bidManager.js [hunk 4]
```diff
@@ -371,6 +374,7 @@ exports.executeMonetizationPixel = function(slotID, theBid){ // TDD, i/o : done
 
 function analyticalPixelCallback(slotID, bmEntry, impressionIDMap) { // TDD, i/o : done
     var startTime = bmEntry.getCreationTime();
+    var pslTime = impressionIDMap.psl;
```
Comment (pm-harshad-mane) [inline src_new/bidManager.js]: There are no Unit test cases for this change.

File: src_new/bidManager.js (hunk 2 above)
Comment (pm-manasi-moghe) [reply src_new/bidManager.js]: fixed it as per eslint in gulpfile.js

File: src_new/bidManager.js (hunk 3 above)
Comment (pm-manasi-moghe) [reply src_new/bidManager.js]: fixed it as per eslint in gulpfile.js

**Pull request merged**